docs/repositories/README.md

```diff
@@ -71,16 +71,18 @@ interface OAuthTokenRepository {
   // An async call that should return an OAuthToken that has not been 
   // persisted to storage yet.
   issueToken(
-    client: OAuthClient, 
-    scopes: OAuthScope[], 
+    client: OAuthClient,
+    scopes: OAuthScope[],
     user?: OAuthUser
   ): Promise<OAuthToken>;
 
-  issueRefreshToken(accessToken: OAuthToken): Promise<OAuthToken>
-
   // An async call that should persist an OAuthToken into your storage.
   persist(accessToken: OAuthToken): Promise<void>;
 
+  // An async call that enhances an already-persisted OAuthToken with
+  // refresh token fields.
+  issueRefreshToken(accessToken: OAuthToken, client: OAuthClient): Promise<OAuthToken>
```

Comment:
I've moved this method down, and given it an explanation, to match its actual usage.
(see e.g. https://github.com/jasonraimondi/ts-oauth2-server/blob/main/src/grants/auth_code.grant.ts#L110:L112, and https://github.com/jasonraimondi/ts-oauth2-server/blob/main/src/grants/abstract/abstract.grant.ts#L215:L229)